package.json

```diff
@@ -10,7 +10,8 @@
     "backbone.paginator": "~2.0.3",
     "coffee-loader": "^0.7.3",
     "coffee-script": "1.6.1",
-    "@edx/studio-frontend": "0.1.0",
+    "css-loader": "^0.28.5",
+    "@edx/studio-frontend": "edx/studio-frontend#27668cd9595eb1f6a6e68fcab3d3ac5273aec50a",
```

Comment:
I plan on changing this to `0.1.2` after https://github.com/edx/studio-frontend/pull/31 is merged and published